README.md

```diff
@@ -1,14 +1,8 @@
 ## Introduction
 
-**ebi-metagenomics/miassembler** is a bioinformatics pipeline for the assembly of short metagenomic reads using [SPAdes](https://doi.org/10.1089/cmb.2012.0021) or [MEGAHIT](https://doi.org/10.1093/bioinformatics/btv033).
+**ebi-metagenomics/miassembler** is a bioinformatics pipeline for the assembly of long and short metagenomic reads.
 
-1. Read QC using [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
-2. Present QC for raw reads and assembly [MultiQC](http://multiqc.info/)
-3. Performs assembly using [MEGAHIT](https://github.com/voutcn/megahit) and [SPAdes](http://cab.spbu.ru/software/spades/), and checks assembly quality using [Quast](http://quast.sourceforge.net/quast)
-4. Removes contaminated contigs using [BLASTN](https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&PAGE_TYPE=Blastdocs) and [SeqKit](https://bioinf.shenwei.me/seqkit/)
-5. Calculates assembly coverage using [MetaBAT2](https://bitbucket.org/berkeleylab/metabat/src/master/) metabat2_jgi_summarizebamcontigdepths for per contig depth and [Samtools idxstats](http://www.htslib.org/doc/samtools-idxstats.html) for alignment summary statistics.
-
-This pipeline is still in early development. It's mostly a direct port of the mi-automation assembly generation pipeline. Some of the bespoke scripts used to remove contaminated contigs or to calculate the coverage of the assembly were replaced with tools provided by the community ([SeqKit](https://doi.org/10.1371/journal.pone.0163962) and [quast](https://doi.org/10.1093/bioinformatics/btu153) respectively).
```

Comment:
I think we should keep and expand this + add a schema at some point. (We can do this afterwards)

Reply:
Sorry, I just saw the README per mode. Please link those here.

```md
This pipeline supports both short and long reads; however, it does not yet support hybrid assemblies.

The steps of the pipeline are outlined in the following documents:

    [Short Reads](README_SHORT_READS.md)
    [Long Reads](README_LONG_READS.md)
```